src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testGVCFMode.gatk4.g.vcf

```diff
@@ -924,14 +923,16 @@
 20	10068200	.	A	<NON_REF>	.	.	END=10068200	GT:DP:GQ:MIN_DP:PL	0/0:16:45:16:0,45,675
 20	10068201	.	T	<NON_REF>	.	.	END=10068201	GT:DP:GQ:MIN_DP:PL	0/0:18:51:18:0,51,765
 20	10068202	.	A	<NON_REF>	.	.	END=10068203	GT:DP:GQ:MIN_DP:PL	0/0:19:54:19:0,54,810
-20	10068204	.	A	<NON_REF>	.	.	END=10068205	GT:DP:GQ:MIN_DP:PL	0/0:21:57:20:0,57,855
-20	10068206	.	C	<NON_REF>	.	.	END=10068206	GT:DP:GQ:MIN_DP:PL	0/0:21:54:21:0,54,810
-20	10068207	.	C	<NON_REF>	.	.	END=10068208	GT:DP:GQ:MIN_DP:PL	0/0:22:57:21:0,57,855
-20	10068209	.	C	<NON_REF>	.	.	END=10068209	GT:DP:GQ:MIN_DP:PL	0/0:23:63:23:0,63,945
-20	10068210	.	T	<NON_REF>	.	.	END=10068219	GT:DP:GQ:MIN_DP:PL	0/0:27:72:26:0,72,1080
-20	10068220	.	C	<NON_REF>	.	.	END=10068224	GT:DP:GQ:MIN_DP:PL	0/0:29:81:29:0,81,1215
-20	10068225	.	G	<NON_REF>	.	.	END=10068225	GT:DP:GQ:MIN_DP:PL	0/0:32:90:32:0,90,1350
-20	10068226	.	T	<NON_REF>	.	.	END=10068237	GT:DP:GQ:MIN_DP:PL	0/0:32:84:30:0,84,1260
+20	10068204	.	A	<NON_REF>	.	.	END=10068206	GT:DP:GQ:MIN_DP:PL	0/0:21:57:20:0,57,855
```

Comment:
Good lord, why do we block GQ50s by ones?!  Ugh!  Your fix probably won't help merge blocks except in cases like this where the GQ is already in the high 50s or the one above where there's a little fluctuation between ranges.  All of these look perfectly reasonable to me.  If you wanted to be super duper rigorous you could temporarily add a test at BP resolution and we could compare and contrast locus by locus to take the blocking variable out.  (I expect the only changes to be increases on the range of 3 in the new version.)

Reply:
It looks like we block everything under 60 by single bases then block by 10's after that. I remember hearing some talk about improving this behavior elsewhere but that sounds like a discussion for another time though. Either way I think this branch will moderately improve our confidence after deletions, even if it is only a small effect.